dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/util/AnalyticsSqlUtils.java

```diff
@@ -237,4 +239,25 @@ public static String getCoalesce( List<String> fields, String defaultColumnName
         return args.isEmpty() ? defaultColumnName
             : "coalesce(" + args + ")";
     }
+
+    /**
+     * This method will simply prefix the given column with the collate
+     * function. ie: columnA -> collate "columnA"
+     *
+     * The final statement is surrounded by blank spaces to make its usage safet
+     * to the caller.
+     *
+     * @param column the column to be "collated".
+     * @return the collate statement, or blank if the given column is
+     *         null/blank.
+     */
+    public static String getCollate( String column )
```

Comment:
I would pass the enum itself here. Also column isn't the proper name for this parameter

Reply:
In this case, I preferred to keep it as String because this is a `util` class. Avoiding dependencies on custom types makes it more reusable I believe...